.env.development

```diff
@@ -69,10 +67,6 @@ APPOINTMENT_NOTIFICATION_HOUR_OF_DAY_FINISH=16
 APPOINTMENT_NOTIFICATION_HOUR_OF_DAY_START=14
 APPOINTMENT_NOTIFICATION_FAN_OUT_BATCH_SIZE=
 
-BACKFILL_USER_ID_FROM_AUDIT_LOGS_BATCH_SIZE=1000
-BACKFILL_ENCOUNTERS_FOR_BPS_BATCH_SIZE=1000
-BACKFILL_ENCOUNTERS_FOR_BPS_TIMEZONE_OFFSET=19800
-
```

Comment:
These are unused config vars (remnant of [data migrations](https://github.com/simpledotorg/simple-server/pull/644))